applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js

```diff
@@ -157,18 +157,31 @@ return view.extend({
 		p[1].default = fwDefaults.getOutput();
 		p[2].default = fwDefaults.getForward();
 
-		o = s.taboption('general', form.Flag, 'masq', _('Masquerading'),
-			_('Enable network address and port translation (NAT or NAPT) for outbound traffic on this zone. This is typically enabled on the <em>wan</em> zone.'));
+		o = s.taboption('general', form.Flag, 'masq', _('IPv4 Masquerading'),
+			_('Enable network address and port translation IPv4 (NAT4 or NAPT4) for outbound traffic on this zone. This is typically enabled on the <em>wan</em> zone.'));
 		o.editable = true;
 		o.tooltip = function(section_id) {
+			var family = uci.get('firewall', section_id, 'family')
 			var masq_src = uci.get('firewall', section_id, 'masq_src')
 			var masq_dest = uci.get('firewall', section_id, 'masq_dest')
-			if (masq_src || masq_dest)
+			if ((!family || family === 'ipv4') && (masq_src || masq_dest))
```

Comment:
The check is overly strict, `family` may be `inet`, `inet6`, `ipv4`, `ipv6`, `4`, `6`. I suggest to do `family.indexOf('6') == -1`

Reply:
Got it, thanks!